flow/connectors/postgres/qrep_sync_method.go

```diff
@@ -81,14 +82,27 @@ func (s *QRepStagingTableSync) SyncQRepRecords(
 		if err != nil {
 			return -1, fmt.Errorf("failed to copy records into destination table: %v", err)
 		}
+
+		if syncedAtCol != "" {
+			updateSyncedAtStmt := fmt.Sprintf(
+				`UPDATE %s SET "%s" = CURRENT_TIMESTAMP WHERE "%s" IS NULL;`,
+				pgx.Identifier{dstTableName.Schema, dstTableName.Table}.Sanitize(),
+				syncedAtCol,
+				syncedAtCol,
+			)
+			_, err = tx.Exec(context.Background(), updateSyncedAtStmt)
+			if err != nil {
+				return -1, fmt.Errorf("failed to update synced_at column: %v", err)
+			}
+		}
 	} else {
 		// Step 2.1: Create a temp staging table
 		stagingTableName := fmt.Sprintf("_peerdb_staging_%s", shared.RandomString(8))
 		stagingTableIdentifier := pgx.Identifier{s.connector.metadataSchema, stagingTableName}
 		dstTableIdentifier := pgx.Identifier{dstTableName.Schema, dstTableName.Table}
 
 		createStagingTableStmt := fmt.Sprintf(
-			"CREATE UNLOGGED TABLE %s (LIKE %s);",
+			"CREATE UNLOGGED TABLE %s (LIKE %s INCLUDING INDEXES);",
```

Comment:
Is this related or just an optimization realized while implementing?

Reply:
It isn't needed. Initially thought it was